CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# v81.0.1 (_2021-07-19_)
+
+[Full Changelog](https://github.com/mozilla/application-services/compare/v81.0.0...v81.0.1)
+
+## Tabs
+### ⚠️ Breaking Changes ⚠️
+    - Tab struct member last_used is now a i64
```

Comment:
You could consider putting a note here about how "we do not expect any consumers to have updated to 81 in practice so we're treating this as a bugfix release" just to be in the mindset of using the changelog to communicate with consumers, but I certainly won't block on that.